src/community/design-system-day/index.md

```diff
@@ -31,18 +31,26 @@ The first event this year will be on 21 May 2024. It will be a free, online even
 
 ### Speakers
 
-**What will the GOV.UK Design System look like in 5 years?**
+**The GOV.UK Design System in 5 years?**
 Trang Erskine, Charlotte Downs & Romaric Pascal, GOV.UK Design System
 
+A show & tell exploring the history of the GOV.UK Design System, and it's focus for the next few years.
```

Comment:
It's vs. its, but also saying "ours" might be friendlier

```suggestion
A show & tell exploring the history of the GOV.UK Design System, and our focus for the next few years.
```

Reply:
Also, GOV.UK Style Guide would tell us to use "and" instead of ampersands "&"...

Reply:
Also, GOV.UK Style Guide would tell us to use "and" instead of ampersands "&"...

```suggestion
A show and tell exploring the history of the GOV.UK Design System, and our focus for the next few years.
```